.github/workflows/ci.yml

```diff
@@ -31,8 +31,8 @@ jobs:
         with:
           node-version: 16.x
           cache: npm
-      - run: make freshdocs
-      - run: node scripts/git-dirty.js
```

Comment:
No docs are checked in so there isn't any need to check git status here anymore.